tools/misbehaving-jmx-server/src/main/java/org/datadog/misbehavingjmxserver/App.java

```diff
@@ -45,13 +45,20 @@ class AppConfig {
     @Parameter(names = {"--rmi-host", "-rh"})
     public String rmiHost = Defaults.JMXSERVER_RMI_INTERFACE;
 
+    @Parameter(names = {"--rng-seed", "-rs"})
+    public Long rngSeed = 54321L;
+
     // Can only be set via env var
     public int controlPort = Defaults.JMXSERVER_CONTROL_PORT;
 
     @Parameter(names = {"--config-path", "-cfp"})
     public String config_path = "./misbehaving-jmx-domains-config.yaml";
 
+    @Parameter(names = {"--seed-config-path", "-scfp"})
+    public String seed_config_path = "./misbehaving-seed-config.yaml";
```

Comment:
Can we integrate this into the existing config file? I don't see a need to separate it into its own config file

Reply:
The old config file was specific to `domains`. I have made changes to only use 1 config file, renamed some of the codenames and just call config file `misbehaving-config.yaml` now. 

Open to suggestions for naming